src/app/components/Block/index.js

```diff
@@ -334,6 +345,8 @@ export const transformSection = section => {
   const [, videoScrollplayPctString] = section.configString.match(SCROLLPLAY_PCT_PATTERN) || [, ''];
   const videoScrollplayPct =
     videoScrollplayPctString.length > 0 && Math.max(0, Math.min(100, +videoScrollplayPctString));
+  const [, blockBackgroundColourString] = section.configString.match(BLOCK_BACKGROUND_COLOUR_PATTERN) || [, '000000'];
```

Comment:
Two points:
1. Does `section.configString` always exist? If it's missing this will throw.
2. Does this default of `000000` make all blocks black by default? Is this desirable? Or would it be better as `00000000` (specifying the alpha channel to set blocks transparent by default)?

Reply:
Good point!

I believe that it does always exist, as line 345 makes that assumption and seems to work in the wild..

I've pushed a change that only applies a background at all if this regex matches.